src/js/common/data_graphic.js

```diff
@@ -89,7 +89,7 @@ MG.data_graphic = function(args) {
     show_rollover_text: true,
     show_confidence_band: null,                 // given [l, u] shows a confidence at each point from l to u
     xax_format: null,                           // xax_format is a function that formats the labels for the x axis.
-    area: true,
+    area: true,                                 // Can be also an array to select lines having areas (e.g. [1, 3])
```

Comment:
I'm wondering what the right API here is. We allow a user to pass in an array of arrays for data. Perhaps this should be an array of booleans. So if my `data` is three arrays, I should have to pass in `area: [false, false, true]`. I am not totally happy with how we are mostly passing these things in positionally, but from a semantic point of view I think this makes sense given how the library works.

Another possibility is that perhaps you can pass a function into array, which maps over each value of the data array (if you have an array of arrays) which can set `area: true` for that particular array.

@wlach what do you think about this from an API point of view?

Reply:
I'll handle that in upcoming days.